### PR TITLE
Remove accounting method from CYA view

### PR DESCRIPTION
**Pull request by ian-ralphs**

### CODE REVIEW

File: test/models/common/subscription/CreateIncomeSourcesModelSpec.scala [hunk 1]
```diff
@@ -35,7 +35,7 @@ class CreateIncomeSourcesModelSpec extends PlaySpec with MustMatchers {
 
   val fullSoleTraderBusinesses: SoleTraderBusinesses = SoleTraderBusinesses(
     accountingPeriod = AccountingPeriodModel(now, now),
-    accountingMethod = Cash,
+    accountingMethod = Some(Cash),
```
Comment (ILee15) [inline test/models/common/subscription/CreateIncomeSourcesModelSpec.scala:38]: I think it would be good to add tests for when `accountingMethod = None` in each income source in this model spec

**Review by ILee15 — APPROVED** [review]
